generate_demog_maps.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import json
+from core_data_modules.cleaners import Codes
+import sys
+
+from core_data_modules.logging import Logger
+from core_data_modules.traced_data.io import TracedDataJsonIO
+
+from src.lib import PipelineConfiguration
+
+Logger.set_project_name("IMAQAL")
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Generates graphs for analysis")
+
+    parser.add_argument("user", help="User launching this program")
+    parser.add_argument("messages_json_input_path", metavar="messages-json-input-path",
+                        help="Path to a directory with JSON files to read the TracedData of the messages data from")
+    parser.add_argument("demog_map_json_output_path", metavar="demog-map-json-output-path",
+                        help="Path to a JSON file to write per episode demog map ")
```

Comment:
It's not a file, it's a directory.

It may also help to rename this variable to `demog_map_json_output_dir`

Reply:
Updated thanks.